third_party/pyth/xc-governance-sdk-js/src/chains.ts

```diff
@@ -0,0 +1,11 @@
+import { CHAINS as WORMHOLE_CHAINS } from "@certusone/wormhole-sdk";
+
+const RECEIVER_CHAINS = {
+  cronos: 60001,
+  kcc: 60002,
+  zksync: 60003,
+};
+
+export const CHAINS = { ...WORMHOLE_CHAINS, ...RECEIVER_CHAINS };
```

Comment:
maybe check here that there aren't any reused chain ids?

Reply:
Hmm.. I double checked that the second term replaces the previous one. So I think it is fine as there might be times that wormhole is there and we haven't migrated yet.

Reply:
Added a comment about that.